.eslintrc.js

```diff
@@ -1,6 +1,9 @@
-module.exports = {
+export default {
   extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended'],
   parser: '@typescript-eslint/parser',
   plugins: ['@typescript-eslint'],
   root: true,
+  rules: {
+    '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
This is strongly discouraged this unless absolutely necessary, in favour of things like `as unknown as <desired type>` for typecasting, and just simply stronger typing overall.

In the case of APIs where any input is necessary, either use generics (`<T,T2,etc...>`).

Reply:
This is so the event callbacks can have the `any[]` type. Using `unknown[]` results in ts errors

Reply:
> This is so the event callbacks can have the `any[]` type. Using `unknown[]` results in ts errors

Generics would allow this to work without any if events were moved to their own class; see below

Reply:
> > This is so the event callbacks can have the `any[]` type. Using `unknown[]` results in ts errors
> 
> Generics would allow this to work without any if events were moved to their own class; see below

To add onto that, even if `any` was the only option, `no-explicit-any` should be disabled on a line-by-line or, at the very least, a file-by-file basis